docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<ul><li>"If user" -&gt; If a user</li><li>"provides just names": what are 'names' in this context?</li><li>"then RP uses it " -&gt; then RP uses them?</li><li>expands that -&gt; expands them?</li><li>"For example, staging directive" -&gt; For example, a staging directive</li><li>"in task description" -&gt; in a task description || in TaskDescription</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>resolved</p>